src/main/java/bio/terra/cli/service/utils/TerraCredentials.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.cli.service.utils;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.IdToken;
+
+/**
+ * Wrapper class to hold a user's credential to use when calling Terra API. Consists of a
```

Comment:
Do you mean the Workspace Manager API?

Reply:
No, any Terra API (WSM API, Sam API, ECM API...).  Is there a better way to word that?

Reply:
maybe "any Terra service API"